sky/data/mounting_utils.py

```diff
@@ -121,8 +206,7 @@ def get_mounting_script(
     Returns:
         str: Mounting script as a str.
     """
-
-    mount_binary = mount_cmd.split()[0]
+    mount_binary = _get_mount_binary(mount_cmd)
```

Comment:
why do we change from `mount_cmd.split()[0]` to `mount_cmd`?

Reply:
You can see `get_s3_mount_cmd`. `mount_cmd.split()[0]` was enough before, but we now have env. vars passed in as well for `blobfuse` so the `0` idx is not the binary anymore.

Reply:
Do you mean this one? seems like the `goofys` is still the first word?

https://github.com/skypilot-org/skypilot/blob/feaaa3a21c16c0d8870f209020dad6de1a009454/sky/data/mounting_utils.py#L27-L33

Reply:
`blobfuse2` wouldn't work for this since there's a number. And I think changing the methods entirely requires additional tests to see if there's any edge case. If there isn't a bug with current approach, let's wait until one is found in the future.